internal/namespaces/rdb/v1/custom_log.go

```diff
@@ -14,3 +18,26 @@ var (
 		rdb.InstanceLogStatusError:    &human.EnumMarshalSpec{Attribute: color.FgRed, Value: "error"},
 	}
 )
+
+func logPrepareBuilder(c *core.Command) *core.Command {
+	c.WaitFunc = func(ctx context.Context, argsI, respI interface{}) (interface{}, error) {
+		getResp := respI.(*rdb.PrepareInstanceLogsResponse)
+		api := rdb.NewAPI(core.ExtractClient(ctx))
+		var readyLogs []*rdb.InstanceLog
+		for i := 0; i < len(getResp.InstanceLogs); i++ {
+			logs, err := api.WaitForInstanceLog(&rdb.WaitForInstanceLogRequest{
+				InstanceLogID: getResp.InstanceLogs[i].ID,
+				Region:        getResp.InstanceLogs[i].Region,
+				Timeout:       scw.TimeDurationPtr(instanceActionTimeout),
+				RetryInterval: core.DefaultRetryInterval,
+			})
+			if err != nil {
+				return nil, err
+			}
+			readyLogs = append(readyLogs, logs)
+		}
```

Comment:
```suggestion
		readyLogs := make([]*rdb.InstanceLog, len(getResp.InstanceLogs))
		for i := range getResp.InstanceLogs {
			logs, err := api.WaitForInstanceLog(&rdb.WaitForInstanceLogRequest{
				InstanceLogID: getResp.InstanceLogs[i].ID,
				Region:        getResp.InstanceLogs[i].Region,
				Timeout:       scw.TimeDurationPtr(instanceActionTimeout),
				RetryInterval: core.DefaultRetryInterval,
			})
			if err != nil {
				return nil, err
			}
			readyLogs[i] = logs
		}
```

If size is known you can allocate the slice.
To use append, it could also be done using capacity instead of size ([gotour](https://go.dev/tour/moretypes/13)).

Reply:
I will keep it in mind for the future :) Thanks, just made the change.